src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -164,6 +187,9 @@ private async Task<SearchResult> SearchImpl(SearchOptions searchOptions, SqlSear
                 }
             }
 
+            var originalSearchOptions = searchOptions;
```

Comment:
>var originalSearchOptions = searchOptions; [](http://example.com/codeflow?start=12&length=42)

since it's uses only `originalSearchOptions.Sort` maybe it should be
`var originalSort = searchOption.Sort`?